packages/wallets/src/components/WalletListCardDropdown/WalletListCardDropdown.scss

```diff
@@ -0,0 +1,62 @@
+.wallets-list-card-dropdown {
+    &__item {
+        display: flex;
+        gap: 1.6rem;
+
+        &-content {
+            display: flex;
+            flex-direction: column;
+        }
+    }
+
+    .wallets-textfield {
+        min-width: unset;
+
+        &__box {
```

Comment:
minor/optional: usually we shouldn't override internal parts of wallet-textfield from different component.
As this creates tight coupling (I'm no longer able to modifiy Textfiled component without modyfying WalletListCardDropdown... which is still doable, but then, if we generate more tight coupling, suddenly, to change small component, one need to refactor half of the application)

Typically the solution is to add extra variant/modifier/parameter to original component, or create new different componnent for different use case, or go back to designers if the issue is caused by design incosistencies.

Reply:
Adding a variant specifically for WalletListCardDropdown sounds like a better approach, will add changes accordingly 👍

Reply:
Updated